### DISCUSSION TITLE
Unable to delete and update data from rowy (sometimes)

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#717] [Support / Q&A] [answered]
I'm not sure what causes this bug. Sometimes it's not possible to delete or update data from rowy, and some others it is.

https://user-images.githubusercontent.com/94030557/179785617-2cadd51e-7575-47b3-8298-449a46b0e9b0.mp4

**Comment by notsidney** [comment, marked as answer]
This is very odd. Could you please check that you set the rules so that you can delete or update documents to this collection in Firebase?

**Reply by st-trade** [reply]
I sure rules are not the problem, since it sometimes allows to update and delete.

**Reply by harinij** [reply]
It appears to be some type of custom rules in place in your project, potentially at row level which makes it possible for you to delete/update some rows and not others. If you check, I am assuming the rows that you are unable to update are the same rows that you can't delete? Since this is custom rules at your project level and outside the scope of Rowy, closing this issue for now. Please reply here if you still think this is an issue.
